core/nilai.go

```diff
@@ -0,0 +1,28 @@
+package core
+
+import (
+	"context"
+
+	"github.com/tokopedia/code-review-workshop/errors"
+	"github.com/tokopedia/code-review-workshop/tracer"
+)
+
+type NilaiSiswa struct {
+	NIM     int64  `json:"nim"      db:"nim"`
+	MapelID int64  `json:"mapel_id" db:"mapel_id"`
+	Nilai   int64  `json:"nili"     db:"nili"`
+	Sekolah string `json:"sekolah"  db:"sekolah"`
+	Kelas   string `json:"kelas"    db:"kelas"`
+}
+
+func GetNilaiSiswaByID(ctx context.Context, nim, mapel_id int64) (result []NilaiSiswa, err error) {
```

Comment:
disarankan pakai camel case ya jd bisa `mapelID`
```suggestion
func GetNilaiSiswaByID(ctx context.Context, nim, mapelID int64) (result []NilaiSiswa, err error) {
```